types/src/front/files.ts

```diff
@@ -0,0 +1,115 @@
+import * as t from "io-ts";
+
+// File upload form validation.
+
+export const FileUploadUrlRequestSchema = t.type({
+  contentType: t.string,
+  fileName: t.string,
+  fileSize: t.number,
+  useCase: t.literal("conversation"),
+});
+
+export type FileUploadUrlRequestType = t.TypeOf<
+  typeof FileUploadUrlRequestSchema
+>;
+
+export interface FileRequestResponseBody {
+  file: FileType;
+}
+
+// Define max sizes for each category.
+const MAX_SIZES: Record<"plainText" | "image", number> = {
+  plainText: 30 * 1024 * 1024, // 30MB.
+  image: 3 * 1024 * 1024, // 3 MB
```

Comment:
I thiiink we can accept a bit more no?

Reply:
We will revisit once we have vision live 🙃.